enginetest/testdata.go

```diff
@@ -65,10 +67,17 @@ func wrapInTransaction(t *testing.T, db sql.Database, harness Harness, fn func()
 	}
 }
 
+func SetSpatialSupported(v bool) {
+	spatialSupported = v
+}
+
 // createSubsetTestData creates test tables and data. Passing a non-nil slice for includedTables will restrict the
 // table creation to just those tables named.
 func CreateSubsetTestData(t *testing.T, harness Harness, includedTables []string) []sql.Database {
 	dbs := harness.NewDatabases("mydb", "foo")
+	if spatialSupported {
```

Comment:
Do not do this

Either a) make the harness itself decide which tables to include in test setup in some way or b) figure out how to make this separable via a distinct test method, as it was before this